### DISCUSSION TITLE
[4.0] Quick Edit on Articles

### DISCUSSION THREAD

**Discussion by itxchandan** [joomla/joomla-cms#24997] [2. Cold Features]
Am very happy with quick change the Article Status. Really this will very helpful.

![image](https://user-images.githubusercontent.com/17069784/58313280-d04dda00-7e2a-11e9-92f9-1f2940148c8a.png)

Same like Article status. If we provide Quick change of **Article Title**, **Category**, **Status**. Will be much easier for Joomla Users.
And If we can Ajax Save. It will more powerful.

**Comment by ghost** [comment]
@bembelimen can you please comment on this Request?

**Comment by bembelimen** [comment]
@Hackwar is working on something like that of I remember correctly.

**Comment by Hackwar** [comment]
No, I'm currently not.

**Comment by itxchandan** [comment]
@franz-wohlkoenig 
Thanks for Edit the Title and Adding the label.

**Comment by itxchandan** [comment]
@Hackwar 
Are we offering this type of feature in Joomla 4?

**Comment by brianteeman** [comment]
I would be very surprised if someone takes the time to code this. It is a big change

**Comment by alikon** [comment]
Closing this due to not receiving required information to determine if this is a bug or not. If you feel this still needs review, please open a new tracker entry with as much information as possible to ensure it can be reviewed properly

**Comment by itxchandan** [comment]
@alikon 
This is not a bug. If you read this post. Am talking about new feature.

**Comment by Bakual** [comment]
Actually, TincyMCE would support inline editing. And with the new API endpoints in J4, I suppose it could be doable.
Personally I think it would be a great idea.

**Comment by itxchandan** [comment]
@Bakual 
Yes, I Agreed
